### PR TITLE
Updated the README.md

### PR DESCRIPTION
**Pull request by yaseminbridges**
Updated the README.md to better introduce the project with some example usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,80 @@
...
+    --phenopacket-dir /phenopackets \
+    --variant-analysis \
+    --gene-analysis \
+    --gene-identifier ensembl_id \
+    --hg19-template-vcf /template_hg19.vcf \
+    --hg38-template-vcf /template_hg38.vcf \
+    --output-dir /vcf
+```
+
+See the [documentation](https://monarch-initiative.github.io/pheval/executing_a_benchmark/) for instructions on benchmarking and evaluating the performance of various VGPAs.
```
Comment (julesjacobsen) [inline README.md:79]: This link leads to a 404 :(

**Review by julesjacobsen — CHANGES_REQUESTED** [review]
Looks great. Can you update or fix the broken link on the last line of the README before merging.

File: README.md (hunk 1 above)
Comment (yaseminbridges) [reply README.md:79]: This link will work once #348 is merged

**Comment by julesjacobsen** [comment]
OK - please merge this once the tox conflict is fixed in #348

**Pull request merged**